test/e2e_node/runner/remote/run_remote.go

```diff
@@ -192,11 +203,12 @@ func main() {
 			}
 			for _, image := range images {
 				gceImage := internalGCEImage{
-					image:    image,
-					project:  imageConfig.Project,
-					metadata: getImageMetadata(imageConfig.Metadata),
-					machine:  imageConfig.Machine,
-					tests:    imageConfig.Tests,
+					image:     image,
+					project:   imageConfig.Project,
+					metadata:  getImageMetadata(imageConfig.Metadata),
+					machine:   imageConfig.Machine,
+					tests:     imageConfig.Tests,
+					resources: &imageConfig.Resources,
```

Comment:
There is a bug here. If we are not using image config file, we'll not go to this line, and `resources` will be left nil.